modules/machine-health-checks-about.adoc

```diff
@@ -6,15 +6,32 @@
 [id="machine-health-checks-about_{context}"]
 = About machine health checks
 
-Machine health checks automatically repair unhealthy machines in a particular machine pool.
+You can define conditions under which machines in a Cluster are considered unhealthy by using a `MachineHealthCheck` resource.
+Machines matching the conditions are automatically remediated.
 
-To monitor machine health, you create a resource to define the configuration for a controller. You set a condition to check for, such as staying in the `NotReady` status for 15 minutes or displaying a permanent condition in the node-problem-detector, and a label for the set of machines to monitor.
+When defining a machine health check, you can specify conditions to check for and set a timeout for each of the conditions.
+A machine will be remediated if any of the conditions are met for the duration of the timeout.
+The action of remediating a machine will trigger a new machine to be created, to replace the failed one.
+
+For example, you may set a condition such as staying in the `NotReady` state for 15 minutes or displaying a permanent condition in the node-problem-detector.
 
 [NOTE]
 ====
-You cannot apply a machine health check to a machine with the master role.
+Please note that machine health checks *only* support machines that are owned by a machine set.
+Please review the <<machine-health-checks-limitations_{context}>> below for full details of machine health check limitations.
```

Comment:
Cross-references are not currently allowed in module files. Please can you remove this line?

Reply:
This one isn't a cross reference is it? It refers to the next section in the module and links to it (@elmiko tested this), I didn't think this should cause an issue?

Reply:
I checked this with @vikram-redhat and this is essentially a cross-reference within the module itself and this is not currently supported.

Reply:
Ok, I've removed it :)

Reply:
Thank you!